server/public/model/job.go

```diff
@@ -78,6 +80,8 @@ type Job struct {
 	Status         string    `json:"status"`
 	Progress       int64     `json:"progress"`
 	Data           StringMap `json:"data"`
+
+	Logger mlog.Sugar `json:"-"`
```

Comment:
Embedding the logger here with context is great.  I would suggest putting a `mlog.Logger` instead of Sugar.  Any job developers can easily turn `mlog.Logger` into a Sugar in their job code, but storing the Sugar means they are stuck with the less structured API.